frontend/utilities/ace_editor.ts

```
@@ -0,0 +1,35 @@
+  editor.container.addEventListener(
+    "wheel",
+    (e: WheelEvent) => {
+      if (mouseHandler?.isMousePressed && !e.buttons) {
+        mouseHandler.releaseMouse?.();
+      }
+    },
+    // Capture phase so this runs before Ace processes the scroll; passive since
+    // we never call preventDefault (avoids a non-passive wheel-listener warning).
+    { capture: true, passive: true }
+  );
+};
```

ESLint

[error] Prefer default export.

(import/prefer-default-export)